inspirehep/modules/workflows/tasks/arxiv.py

```diff
@@ -122,24 +124,28 @@ def arxiv_plot_extract(obj, eng):
     tarball = obj.files[filename]
 
     if tarball:
+        scratch_space = mkdtemp('plot_extract')
```

Comment:
I discovered we can globally set what root directory to use :+1: by simply setting: 
```python
import tempfile
tempfile.tempdir = '/bla'
```

in the config file.

Reply:
It also respects the env vars 'TMPDIR' and similar.